presto-main/pom.xml

```diff
@@ -412,6 +412,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <argLine>-Xmx4G</argLine>
```

Comment:
Is this limit for BenchmarkPartitionedOutputOperator? There is already a -Xmx10g limit for it.

Reply:
@wenleix  Travis is hitting this error:
[ERROR] Failures: 
[ERROR]   BenchmarkPartitionedOutputOperator_jmhType.verify Â» OutOfMemory Java heap spac...
[ERROR]   BenchmarkPartitionedOutputOperator_jmhType_B2.verify Â» OutOfMemory Java heap s...

Reply:
Looks like `Xmx10g` only applies for benchmark, not for tests :(